source/adapters/level_zero/command_buffer.cpp

```diff
@@ -74,6 +87,9 @@ ur_exp_command_buffer_handle_t_::~ur_exp_command_buffer_handle_t_() {
   if (ZeCommandList) {
```

Comment:
I'm not 100% about this, but wondering if we should rename `ZeCommandList` to `ZeComputeCommandList` everywhere

Reply:
I think this makes sense so I'e made the change.